spec/integration/posts_show_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+describe 'Posts Show', type: :system do
+  let!(:author) { User.create(name: 'Akai321', photo: 'https://Akaiiii.jpg', bio: 'Martial Artist') }
+  let!(:user) { User.create(name: 'Kenshiro', photo: 'https://Kenshiro.jpg', bio: 'Martial Artist') }
+  let!(:post) do
+    Post.create(
+      author:,
+      title: 'The way of the Water',
+      text: 'In martial arts, the way of the water is the way of the soft and yielding.'
+    )
+  end
+  let!(:comment1) do
+    Comment.create(
+      user:,
+      post:,
+      text: 'I prefer the way of the leaf, which is the way of the ninja.'
+    )
+  end
+  let!(:comment2) do
+    Comment.create(
+      user: author,
+      post:,
+      text: 'The way of the leaf is the way of the coward.'
+    )
```

Comment:
Good job, writing some of the required integrations, however, there are many more you've skipped. I recommend that you implement these specs in order to satisfy the milestone's requirement:

- Tests that asserts you can see the comment each commentor left.
- Tests that asserts you can see the username of each commentor.